From 02e71265941b8a162af77fbe35fa08ab2adb08af Mon Sep 17 00:00:00 2001
From: Nicholas Kazlauskas <nicholas.kazlauskas@amd.com>
Date: Mon, 28 Jan 2019 09:41:53 -0500
Subject: [PATCH 1167/2940] Revert "test patch for hpd dpms check"

This reverts commit e29bb93a89d5ac5c0c63a60d83bba22f6c1946d7.

Pushed by accident along with another patch, please ignore.

Change-Id: I1daa19b8d437ba7ee5116ec199aa8bc8850a3330
Signed-off-by: Nicholas Kazlauskas <nicholas.kazlauskas@amd.com>
---
 drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm.c | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm.c b/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm.c
index 5215e0e58da1..6cd479615269 100644
--- a/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm.c
+++ b/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm.c
@@ -1179,9 +1179,6 @@ static void handle_hpd_irq(void *param)
 	if (aconnector->fake_enable)
 		aconnector->fake_enable = false;
 
-	if (aconnector->base.dpms != DRM_MODE_DPMS_ON)
-		goto unlock;
-
 	if (!dc_link_detect_sink(aconnector->dc_link, &new_connection_type))
 		DRM_ERROR("KMS: Failed to detect connector\n");
 
@@ -1207,8 +1204,6 @@ static void handle_hpd_irq(void *param)
 		if (aconnector->base.force == DRM_FORCE_UNSPECIFIED)
 			drm_kms_helper_hotplug_event(dev);
 	}
-
-unlock:
 	mutex_unlock(&aconnector->hpd_lock);
 
 }
-- 
2.17.1

